### DISCUSSION TITLE
TOC in index.html?

### DISCUSSION THREAD

**Discussion by ericthomas1** [mkdocs/mkdocs#3100] [Q&A] [answered]
Instead of all of the directories in my `docs` folder showing in the mkdocs header, how do I instead create a left nav inside `index.md`?

**Example**:
- This is what I want to fix. Its already wrapping and I may add more directories within `docs`
    - ![image](https://user-images.githubusercontent.com/42788729/211893145-2a2ac1f8-125a-4133-92b3-cfe480e39764.png)
- Instead, I'd something like this:
    - ![image](https://user-images.githubusercontent.com/42788729/211894791-9d88fbe7-375c-42c8-a05c-38409e2635c9.png)

Where the left nav can scale beyond what the header can hold. Ideally, the left nav is nestable where user can expand to see files held within each section and navigate directly to the desired page.

How can this be done?

**Comment by oprypin** [comment, marked as answer]
You can just use a different theme.
Most popular would be `readthedocs` (included) or https://squidfunk.github.io/mkdocs-material/

**Reply by ericthomas1** [reply]
I messed with `mkdocs-material`. It looks nice, but was too complex for my use case. Also, I don't want to install another `pip` package. 

I'll try the `readthedocs` theme. 

Thank you

**Reply by ericthomas1** [reply]
The `readthedocs` template worked great. I like that its included in mkdocs.
